host_core/lib/host_core/actors/actor_module.ex

```diff
@@ -336,6 +341,40 @@ defmodule HostCore.Actors.ActorModule do
     HostCore.Refmaps.Manager.put_refmap(oci, pk)
   end
 
+  defp publish_invocation_result(inv, inv_r) do
+    prefix = HostCore.Host.lattice_prefix()
+
+    origin = inv["origin"]
+    target = inv["target"]
+
+    evt_type =
+      if Map.get(inv_r, "error") == nil do
+        "invocation_succeeded"
+      else
+        "invocation_failed"
+      end
+
+    msg =
+      %{
+        source: %{
+          public_key: origin["public_key"],
+          contract_id: Map.get(origin, "contract_id"),
+          link_name: Map.get(origin, "link_name")
+        },
+        dest: %{
+          public_key: target["public_key"],
+          contract_id: Map.get(target, "contract_id"),
+          link_name: Map.get(target, "link_name")
+        },
+        operation: inv["operation"],
+        bytes: byte_size(inv["msg"])
```

Comment:
This looks like it will fail if `inv["msg"]` is `nil`, we might want to do a `Map.get(inv, "msg", "")` here
```
byte_size(nil)
** (ArgumentError) errors were found at the given arguments:

  * 1st argument: not a bitstring

    :erlang.byte_size(nil)
```